bin/sass

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env node
+
+const child_process = require("child_process");
+const {compilerCommand} = require("../dist/lib/src/compiler-path");
+
+// TODO npm/cmd-shim#152 and yarnpkg/berry#6422 - If and when the package
+// managers support it, we should make this a proper shell script rather than a
+// JS wrapper.
+
+try {
+child_process.execFileSync(
+  compilerCommand[0], [...compilerCommand.slice(1), ...process.argv.slice(2)], {
+    stdio: 'inherit',
+    windowsHide: true
+});
```

Comment:
Indent once?

Reply:
This made me realize there's not a great way to get this file covered by the linter/formatter as-is, so I converted it to TypeScript.